assets/js/blocks/checkout/inner-blocks/checkout-order-summary-shipping/block.tsx

```diff
@@ -26,6 +26,7 @@ const Block = ( {
 				showRateSelector={ false }
 				values={ cartTotals }
 				currency={ totalsCurrency }
+				isCheckout={ true }
```

Comment:
What was this for?

Reply:
I found a bug on the checkout block page that displays `Calculated during checkout` for shipping in the Checkout summary if shipping options are unavailable. It happened because we were not passing the `isCheckout` prop to [TotalsShipping component](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/blocks/checkout/inner-blocks/checkout-order-summary-shipping/block.tsx#L24) for Checkout.  This change will fix it and display the correct error message.
![image](https://user-images.githubusercontent.com/11503784/215052665-17b8f1ad-7b33-43be-a689-cf2c356d345a.png)